tools/ci/tc/tasks/test.yml

```diff
@@ -112,6 +112,8 @@ components:
     privileged: true
     scopes:
       - "docker-worker:capability:privileged"
+    chunks-override:
```

Comment:
Can't we just use `chunks` here? That would be similar to what we have in our Taskcluster config on mozilla-central, just with the difference that here it can be set for specific test types.

Reply:
Not easily. We combine everything into a single dictionary of properties for each job, so this would either be overwritten by the chunks specification on the test types, or overwrite it for all test types.